src/components/Settings.vue

```diff
@@ -320,6 +320,26 @@
                           persistent-hint
                         ></v-switch>
                       </v-flex>
+                      <v-flex xs6 v-if="gateway.hassDiscovery">
+                        <v-text-field
+                          v-on="on"
+                          v-model="gateway.entityTemplate"
+                          label="Entity name template"
+                          persistent-hint
+                          hint="Template which generates entity names"
+                        ></v-text-field>
+                      </v-flex>
+                      <v-flex xs6 v-if="gateway.hassDiscovery">
+                        <div>
+                          Default: <code>%loc-%n_%ob</code> <br />-
+                          <code>%n</code>: Node Name <br />- <code>%loc</code>:
+                          Node Location <br />- <code>%pk</code>: valueId
+                          property key (uses property name if undefined) <br />-
+                          <code>%pn</code>: valueId property name <br />-
+                          <code>%ob</code>: HASS object_id <br />-
```

Comment:
I would use `%o`

Reply:
the purpose of using at least 2 letters is to avoid breaking changes in the future. if we introduce another placeholder starting with o. we will have to think priorities of replacement etc.

Reply:
I agree! Good point :)

Reply:
@billiaz We could make this more strict by adding \b at the end of the regex so this will not be a problem

Reply:
makes sense, I do not know if \b reads _ and - .. 
I checked with a regex and seems to work I will update patterns 


For instance from `%nn` to `%\bnn\b`

Reply:
humm ok keep this for now so, no problem. Additionally we could firstly replace the longer occurance and then the smaller ones. For example if we would have `%n` and `%nn` we can firstly replace `%nn` and then `%n`

Reply:
I will revert the %nn to %n, I will add a comment to be careful with conflicting changes and order of replacement

this way I can revert the label to %l too

Reply:
@robertsLando check the new shortened ones. I could replace the l and make it location which makes it look: `%l-%n_%o` and keep label as `lbl` instead of `l`